docs/source/services/timestamp_service/overview.rst

```diff
@@ -0,0 +1,51 @@
+========
+Overview
+========
+
+The Timestamp Service exposes an API that allows for clients to request fresh timestamps, which are required by the
+AtlasDB transaction protocol. These timestamps are *logical timestamps* (that is, they have no relationship to actual
+wall-clock time), and are returned as signed 64-bit integers.
+
+Guarantees
+==========
+
+We guarantee that a request for a fresh timestamp will return a strictly greater timestamp than any other timestamp
```

Comment:
Regarding the guarantee: I've changed it to read "we guarantee that a request for a fresh timestamp will return a strictly greater timestamp than any other timestamp
**already returned** by an earlier request." Hope that's clearer.

I've added an example of a 'bad' sequence of events that we guarantee will not happen, as well.